### DISCUSSION TITLE
[Lesson 42] account default does not exist when deploying on the Testnet

### DISCUSSION THREAD

**Discussion by tcrin** [Cyfrin/foundry-full-course-cu#3012] [Q&A] [answered]
Hi everyone,

I'm deploying a smart contract onto a testnet, using a Makefile for automation. According to the video, my `--account default` does not exist.

My Makefile:
```bash
-include .env

.PHONY: all test clean deploy fund help install snapshot format anvil

build:; forge build

test:; forge test

install :; forge install cyfrin/foundry-devops@0.2.2 --no-commit && forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit && forge install foundry-rs/forge-std@v1.8.2 --no-commit && forge install transmissions11/solmate@v6 --no-commit

deploy-sepolia:
	@forge script script/DeployRaffle.s.sol:DeployRaffle --rpc-url $(SEPOLIA_RPC_URL) --account default --broadcast --verify --etherscan-api-key $(ETHERSCAN_API_KEY) -vvvv
```
When running `make deploy-sepolia`, I receive the following error:
```bash
~/foundry-f23/foundry-smart-contract-lottery$ make deploy-sepolia
Error: 
Keystore file `"/home/xyz/.foundry/keystores/default"` does not exist
make: *** [Makefile:12: deploy-sepolia] Error 1
```
❓**How can I fix the error `Keystore file "/home/xyz/.foundry/keystores/default" does not exist` when deploying a smart contract on the Sepolia network using Foundry?**

**Comment by EngrPips** [comment, marked as answer]
Hello @tcrin, you need to have first encrypted your private key using Foundry `cast wallet import` as specified in the docs https://book.getfoundry.sh/reference/cast/cast-wallet-import before you can then use `--account default` to request it from Foundry i your terminal command.

**Reply by tcrin** [reply]
Thank you @EngrPips, it’s really helpful to me

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.
